### PR TITLE
feat(pitr): pitr support check disk space before do backup and restore

### PR DESCRIPTION
**Pull request by Xu-Wentao**
### Type of change:

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### What this PR does / why we need it:

- `pitr agent` support `api/diskspace` to get local disk space info.
- `pitr cli` support to print all storage nodes disk space info before do backup and restore, and need User to approve to exec tasks.

the whole progress may like this:
```bash
2023-04-25T13:32:40+08:00  info  Checking agent server status...
+---------------------------------+
| Agent Server Status             |
+---+-----------------+-----------+
| # | AGENT SERVER IP | STATUS    |
+---+-----------------+-----------+
| 1 | 127.0.0.1       | Available |
+---+-----------------+-----------+
2023-04-25T13:32:40+08:00  info  Checking disk space...
+--------------------------------------------------------------------------------------+
| Disk Space Status                                                                    |
+---+--------------+----------------+--------------------------------------------------+
| # | DATA NODE IP | DISK PATH      | DISK SPACE STATUS                                |
+---+--------------+----------------+--------------------------------------------------+
| 1 | 127.0.0.1    | /home/omm/data | Filesystem      Size  Used Avail Use% Mounted on |
|   |              |                | /dev/vda1        20G   13G  7.7G  62% /          |
|   |              |                |                                                  |
+---+--------------+----------------+--------------------------------------------------+
Please Check All Nodes Disk Space, Make Sure Have Enough Space To Backup Or Restore Data.
Are you sure to continue? (Y/N)
N
```

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?

### CODE REVIEW

File: pitr/cli/internal/cmd/root.go [hunk 1]
```diff
@@ -56,3 +63,99 @@ var RootCmd = &cobra.Command{
 		HiddenDefaultCmd:  true,
 	},
 }
+
+func getUserApproveInTerminal(prompt string) error {
+	fmt.Println(prompt)
```
Comment (wbtlb) [inline pitr/cli/internal/cmd/root.go:68]: remove debug info

**Review by wbtlb — DISMISSED** [review]

File: pitr/cli/internal/cmd/root.go (hunk 1 above)
Comment (Xu-Wentao) [reply pitr/cli/internal/cmd/root.go]: i use logging.warn to replace it, this place is used to print some Message for user to check and need user input Y|N to continue.

**Review by wbtlb — APPROVED** [review]

**Comment by wbtlb** [comment]
approved

**Review by wbtlb — APPROVED** [review]
approve

**Review by mlycore — APPROVED** [review]

**Pull request merged**